src/app/parcels/ParcelsPage.tsx

```diff
@@ -458,35 +459,33 @@ const ParcelsPage: React.FC<{}> = () => {
 
     useEffect(() => {
         if (!areFiltersLoadingForFirstTime) {
-            const allFilters = [...primaryFilters, ...additionalFilters];
-            const initialRequestParams: RequestParams<ParcelsTableRow> = {
-                allFilters: { ...allFilters },
-                sortState: { ...sortState },
-                startPoint: startPoint,
-                endPoint: endPoint,
-            };
-            (async () => {
-                setIsLoading(true);
-                const filteredParcelCount = await getParcelsCount(supabase, allFilters);
-                const fetchedData = await getParcelsData(
+            const allFilters = [...primaryFilters.slice(), ...additionalFilters.slice()];
+            setIsLoading(true);
+            if (parcelsTableFetchAbortController.current) {
+                parcelsTableFetchAbortController.current.abort("stale request");
+            }
+            parcelsTableFetchAbortController.current = new AbortController();
+            if (parcelsTableFetchAbortController.current) {
+                getParcelsCount(
+                    supabase,
+                    allFilters,
+                    parcelsTableFetchAbortController.current.signal
+                )
+                    .then((count) => setFilteredParcelCount(count))
+                    .catch();
```

Comment:
should we display set any error message in these catches?

Reply:
I don't think so because it could be an aborted request, in which case we don't want to set an error